### PR TITLE
Deprecate VBox/HBox.

### LINKED ISSUE #2010: deprecate VBox/HBox in plotting and charts

### PR DESCRIPTION
**Pull request by Maggie-M**
issues: closes #2010

### CODE REVIEW

File: bokeh/charts/__init__.py [hunk 1]
```diff
@@ -15,7 +16,22 @@
 from ._chart import Chart
 from ._data_adapter import DataAdapter
 
+from ..deprecate import deprecated
 from ..models import ColumnDataSource
 from ..io import (
     curdoc, cursession, output_file, output_notebook, output_server, push,
     reset_output, save, show, gridplot, vplot, hplot)
+
+@deprecated("Bokeh 0.8.2", "bokeh.plotting.vplot function")
```
Comment (bryevdv) [inline bokeh/charts/__init__.py]: better if it says use "bokeh.charts.vplot function" here

File: bokeh/charts/__init__.py [hunk 2]
```diff
@@ -15,7 +16,22 @@
...
+def VBox(*args, **kwargs):
+    ''' Generate a layout that arranges several subplots vertically.
+    '''
+
+    return vplot(*args, **kwargs)
+
+@deprecated("Bokeh 0.8.2", "bokeh.plotting.hplot function")
```
Comment (bryevdv) [inline bokeh/charts/__init__.py]: same comment

**Comment by bryevdv** [comment]
tiny comments, otherwise LGTM

File: bokeh/charts/__init__.py (hunk 1 above)
Comment (Maggie-M) [inline bokeh/charts/__init__.py]: Ah, I'd completely overlooked that.  Thanks!

**Comment by bryevdv** [comment]
Now that I think about we should update the examples in this PR too. All the ones in `examples/plotting` that have things like `show(VBox(p1, p2, p3, p4))` should be updated to import `vplot` instead and do `show(vplot(p1, p2, p3, p4))`

**Pull request merged**